net/mini_snmpd/Makefile

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (C) 2009-2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mini_snmpd
+PKG_VERSION:=1.4-rc1
+PKG_RELEASE:=3
+PKG_MAINTAINER:=Luke McKee <hojuruku@gmail.com>
+PKG_LICENSE:=GPL-2.0
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/troglobit/mini-snmpd.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=203d92e60ed09466d6676c6ad20ad6cb2ce08a5d
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+
+PKG_FIXUP:=autoreconf
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+# As warned by upstream maintainer and gcc
+PKG_SSP:=0
+PKG_RELRO:=0
```

Comment:
I told you I tested without that. Without PKG_RELRO broke the compile, so I blocked it. I added PKG_SSP at the same time I made that change because upstream suggested not to.

I'll comment out PKG_SSP and test now, but PKG_RELRO stays.